qiskit/qobj/pulse_qobj.py

```diff
@@ -0,0 +1,610 @@
+            if self.to_dict() == other.to_dict():
+                return True
+        return False
+
+
+class PulseQobjExperimentConfig:
+    """A config for a single Pulse experiment in the qobj."""
+
+    _data = {}
+
+    def __init__(self, qubit_lo_freq=None, meas_lo_freq=None):
+        """Instantiate a PulseQobjExperimentConfig object.
+
+        Args:
+            qubit_lo_freq (list): List of frequencies (as floats) for the qubit
+                driver LO's in GHz.
+            meas_lo_freq (list): List of frequencies (as floats) for the'
+                measurement driver LO's in GHz.
+        """
+        super(PulseQobjExperimentConfig, self).__init__()
+        self._data = {}
+        if qubit_lo_freq is not None:
+            self._data['qubit_lo_freq'] = qubit_lo_freq
+        if meas_lo_freq is not None:
+            self._data['meas_lo_freq'] = meas_lo_freq
+
+    def to_dict(self):
+        """Return a dictionary format representation of the experiment config.
+
+        Returns:
+            dict: The dictionary form of the PulseQobjExperimentConfig.
+        """
+        return self._data
+
+    @classmethod
+    def from_dict(cls, data):
+        """Create a new PulseQobjExperimentConfig object from a dictionary.
+
+        Args:
+            data (dict): A dictionary for the experiment config
+
+        Returns:
+            PulseQobjExperimentConfig: The object from the input dictionary.
+        """
+        return cls(**data)
+
+    def __eq__(self, other):
+        if isinstance(other, PulseQobjExperimentConfig):
+            if self.to_dict() == other.to_dict():
+                return True
+        return False
+
+
+class PulseLibraryItem:
+    """An item in a pulse library."""
+
+    def __init__(self, name, samples):
+        """Instantiate a pulse library item.
+
+        Args:
+            name (str): A name for the pulse.
+            samples (list[complex]): A list of complex values defining pulse
+                shape.
+        """
+        self.name = name
+        self.samples = samples
+
+    def to_dict(self):
+        """Return a dictionary format representation of the pulse library item.
+
+        Returns:
+            dict: The dictionary form of the PulseLibraryItem.
+        """
+        return {'name': self.name, 'samples': self.samples}
+
+    @classmethod
+    def from_dict(cls, data):
+        """Create a new PulseLibraryItem object from a dictionary.
+
+        Args:
+            data (dict): A dictionary for the experiment config
+
+        Returns:
+            PulseLibraryItem: The object from the input dictionary.
+        """
+        return cls(**data)
+
+    def __eq__(self, other):
+        if isinstance(other, PulseLibraryItem):
+            if self.to_dict() == other.to_dict():
+                return True
+        return False
+
+
+class PulseQobj:
+    """A Pulse Qobj."""
+
+    def __init__(self, qobj_id, config, experiments=None,
```

Comment:
Experiments is a required parameter.

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/3383/commits/a2ff39b1b10ff9262a12990b2c00aa6c611b079e